strings/base_string.h

```diff
@@ -233,7 +233,7 @@ WINRT_EXPORT namespace winrt
             }
             else
             {
-                return {};
+                return { L"", 0 };
```

Comment:
When the `hstring` is empty, the `wstring_view`'s default constructor causes the `data` member to return a `nullptr` as per the spec: 

https://en.cppreference.com/w/cpp/string/basic_string_view/basic_string_view

Reply:
Specs? who needs em!